akka-sample-persistence-typed-scala/src/main/scala/sample/persistence/Main.scala

```diff
@@ -0,0 +1,38 @@
+package sample.persistence
+
+import scala.concurrent.duration._
+
+import akka.actor.typed.ActorRef
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.util.Timeout
+
+object Main extends App {
+implicit val timeout: Timeout = 5.seconds
+
+    case class Ack(description: String)
+
+    // First test: start a single persistent actor.
+    // TODO: start them dynamically, actually do something with responses beyond printing
+    // You can 'manually' test things are actually persisted by changing the commands and looking
+    // at this
+    val rootBehavior = Behaviors.setup[Ack](context => {
+        val persistentActor = context.spawn(ShoppingCart.behavior("test"), "test")
+        context.ask(persistentActor)((self: ActorRef[ShoppingCart.Result]) => ShoppingCart.UpdateItem("foo", 32, self)) {
+            case r => Ack(r.toString)
+        }
+        context.ask(persistentActor)((self: ActorRef[ShoppingCart.Result]) => ShoppingCart.UpdateItem("bar", 12, self)) {
```

Comment:
is the type annotation needed here? try from sbt with Scala 2.12.8

Reply:
(made it a little better though still pretty verboses)